polkadot-parachain/Cargo.toml

```
@@ -27,7 +27,7 @@ statemint-runtime = { path = "../parachain-runtimes/assets/statemint" }
 statemine-runtime = { path = "../parachain-runtimes/assets/statemine" }
 westmint-runtime = { path = "../parachain-runtimes/assets/westmint" }
 canvas-kusama-runtime = { path = "../parachain-runtimes/smart-contracts/canvas-kusama" }
-jsonrpsee = { version = "0.13.0", features = ["server"] }
+jsonrpsee = { version = "0.13.1", features = ["server"] }
```

```suggestion
jsonrpsee = { version = "0.13", features = ["server"] }
```
Would that not be simpler ? Do we really want the `0.13.1` and not the `0.13.2` (for instance) ?

---

this is doesn't really matter `jsonrpsee 0.13.1` is just a doc fix on doc.rs :)

---

My comment is also about future versions. Any `0.13.x` should be compatible so we should take the latest available inseatd of pinning to `0.13.1`. `jsonrpsee` follows semver @niklasad1 right ?

---

yes